delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/OrganizationsRoutes.scala

```diff
@@ -52,9 +53,17 @@ final class OrganizationsRoutes(identities: Identities, organizations: Organizat
   import baseUri.prefixSegment
   implicit val orgContext: ContextValue = Organization.context
 
-  private def orgsSearchParams: Directive1[OrganizationSearchParams] =
-    searchParams.tmap { case (deprecated, rev, createdBy, updatedBy) =>
-      OrganizationSearchParams(deprecated, rev, createdBy, updatedBy)
+  private def orgsSearchParams(implicit caller: Caller): Directive1[OrganizationSearchParams] =
+    searchParams.tflatMap { case (deprecated, rev, createdBy, updatedBy) =>
+      onSuccess(acls.listSelf(AnyOrganization(true)).runToFuture).map { aclsCol =>
```

Comment:
How was working the search before ? If a user has no orgs/read, we returned a 403, a 404 or a empty list with a 200 ?

Reply:
If user has no `org/read` anywhere we return empty list.